core/test/functional/routes/api/upload_icon_spec.js

```diff
@@ -151,7 +151,7 @@ describe('Upload Icon API', function () {
                 .set('Authorization', 'Bearer ' + accesstoken)
                 .expect('Content-Type', /json/)
                 .attach('uploadimage', path.join(__dirname, '/../../../utils/fixtures/images/favicon_size_too_large.png'))
-                .expect(413)
+                .expect(422)
```

Comment:
This is related to changing the error type to `ValidationError`.  If a 413 is really what is needed here, then I'd revert those changes and need a way to change the error message in the client.